Resources/views/standard_layout.html.twig

```diff
@@ -312,6 +315,60 @@ file that was distributed with this source code.
                     </section>
                 {% endblock sonata_page_content %}
             </div>
+
+            {% block sonata_right_side %}
+                <!-- Control Sidebar -->
+                <aside class="control-sidebar control-sidebar-dark">
+                    <!-- Create the tabs -->
+                    <ul class="nav nav-tabs nav-justified control-sidebar-tabs">
+                        <li class="active"><a href="#control-sidebar-home-tab" data-toggle="tab"><i class="fa fa-home"></i></a></li>
+
+                        <li><a href="#control-sidebar-settings-tab" data-toggle="tab"><i class="fa fa-gears"></i></a></li>
+                    </ul>
+                    <!-- Tab panes -->
+                    <div class="tab-content">
+                        <!-- Home tab content -->
+                        <div class="tab-pane active" id="control-sidebar-home-tab">
+                            <h3 class="control-sidebar-heading">{{ 'home_tab'|trans({}, 'SonataAdminBundle') }}</h3>
+                            <ul class='control-sidebar-menu'>
+                                <!-- <li>
```

Comment:
Just copy code as sample is not good. Please cleanup code of those useless comments and give us some use cases.